test/parallel/test-parse-args.mjs

```diff
@@ -0,0 +1,358 @@
+import '../common/index.mjs';
+import assert from 'assert';
+import { test } from 'node:test';
+import { parseArgs } from 'node:parse_args';
+
+// The use of `-` as a positional is specifically mentioned in the Open Group Utility Conventions.
+// The interpretation is up to the utility, and for a file positional (operand) the examples are
+// '-' may stand for standard input (or standard output), or for a file named -.
+// https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html
+//
+// A different usage and example is `git switch -` to switch back to the previous branch.
+test("dash: when args include '-' used as positional then result has '-' in positionals", () => {
```

Comment:
No need to change it if you'd prefer not to, but it looks like some of the tests use arrow functions and some use normal functions.

Reply:
(👍 The newer tests use arrow functions and we haven't yet come back to make a high-churn change to the old tests.)

Reply:
fixed here https://github.com/pkgjs/parseargs/pull/96, will update this PR soon.